hamocc/mo_read_oafx.F90

```diff
@@ -268,10 +286,28 @@ subroutine get_oafx(kpie,kpje,kplyear,kplmon,omask,oafx)
   !--------------------------------
   ! Scenarios of constant fluxes
   !--------------------------------
-  if( trim(oalkscen)=='const_0p14' .or. trim(oalkscen)=='const_0p56' ) then
+  if( trim(oalkscen)=='const' ) then
       
     oafx(:,:) = oalkflx(:,:)
 
+  !--------------------------------
+  ! Scenario of ramping-up fluxes
+  !--------------------------------
+  elseif(trim(oalkscen)=='ramp' ) then
+
+    if(kplyear.lt.ramp_start ) then
+      oafx(:,:) = 0.0
+    elseif(kplyear.ge.ramp_end ) then
+      oafx(:,:) = oalkflx(:,:)
+    else
+      current_day = (kplyear-ramp_start)*365.+nday_of_year
+      oafx(:,:) = oalkflx(:,:) * current_day / ((ramp_end-ramp_start)*365.)
+    endif
+
+    if(mnproc.eq.138 ) then
```

Comment:
This should not be here, sorry. It was for debugging purpose. I was using proc 138 because proc 1 does not include ocean grid cells so I was not able to verify that the code was working. I will remove it.